Phase1/FullTextSearch.java

```diff
@@ -2,29 +2,42 @@
 
 public class FullTextSearch {
     private String command;
+    private InvertedIndex invertedIndex;
 
     public FullTextSearch(String command) {
         this.command = command;
+        invertedIndex = new InvertedIndex();
```

Comment:
instead of get new object in constructor you can just pass it as parameter of constructor, I mean get this new object on main function

Reply:
I don't get it, Could you explain me?

Reply:
Sure! as I saw you called this constructor in another method. I mean is better two get new "InvertedIndex" object in that situation; because as I know it's better to get new object in higher layers as it's possible.

Reply:
اینجا هم خانم ختن لو درست میگن